src/hooks/useUserTenantPermissions/useUserTenantPermissions.js

```diff
@@ -0,0 +1,59 @@
+import { useQuery } from 'react-query';
+
+import {
+  useNamespace,
+  useOkapiKy,
+} from '@folio/stripes/core';
+
+import { OKAPI_TENANT_HEADER } from '../../constants';
+
+const INITIAL_DATA = [];
+
+const useUserTenantPermissions = (
```

Comment:
There is a similar hook `useConsortiumPermissions` in `ui-users` and `ui-consortia-settings`.
In `ui-consortia-settings` they put it in `src/temp` directory to indicate that it's copied code that will be remove later
Let's do the same here

Reply:
The `useConsortiumPermissions` hook returns only consortia permissions, but the ordinary permission needs to be checked. That's why I use the `useUserTenantPermissions` hook.